### PR TITLE
Update to new release.

### PR DESCRIPTION
**Pull request by arokem**

### CODE REVIEW

**Comment by skoudoro** [comment]
Hi @arokem, it seems that appveyor failed because of `conda`. Do you know why ?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 package:
   name: dipy
   version: {{ version }}
 
 source:
   fn: dipy-{{ version }}.tar.gz
-  url: https://pypi.python.org/packages/04/dd/08f9a35a8351d32c1f659821dd72c0549954c126a3d68cc17002ae1dff07/dipy-{{ version }}.tar.gz
+  url:
+    https://pypi.python.org/packages/26/52/a64d196965ec1455371347684a910d3f62aee1d3dac07d00cc68ea5230d0/dipy-{{ version }}.tar.gz
   md5: 72a1d734f7b9545fac46e720e157ac79
```
Comment (skoudoro) [inline recipe/meta.yaml]: Hi @arokem, Can you update the md5? it should work after that.

**Comment by skoudoro** [comment]
2 builds on appveyor failed, should we merge @arokem ? I saw you did it on the previous PR despite this problem.

**Comment by arokem** [comment]
Yeah - I think that we came to the conclusion that we don't know what to do for those cases.

**Pull request merged**